modules/virt-metro-dr-odf.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * virt/backup_restore/virt-disaster-recovery.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="metro-dr-odf_{context}"]
+= Metro-DR for {rh-storage-first}
+
+{VirtProductName} supports the link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.14/html-single/configuring_openshift_data_foundation_disaster_recovery_for_openshift_workloads/index#metro-dr-solution[Metro-DR solution for {rh-storage}], which provides two-way synchronous data replication between managed {VirtProductName} clusters installed on primary and secondary sites. This solution combines {rh-rhacm-first}, Red Hat Ceph Storage, and {rh-storage} components.
+
+Use this solution during a site disaster to fail applications from the primary to the secondary site, and to relocate the application back to the primary site after restoring the disaster site.
+
+This synchronous solution is only available to metropolitan distance data centers with a 10 millisecond latency or less.
+
+For more information about using the Metro-DR solution for {rh-storage} with {VirtProductName}, see link:https://access.redhat.com/articles/7053115[the Red Hat Knowledgebase] or IBM’s {rh-storage} Metro-DR documentation.
```

Comment:
**question**: Should we add the link to IBM documentation as well?

Reply:
@aglitke Do we have a link to the IBM docs?

Reply:
In in the interest of getting this in the docs, leaving as-is. Please open a new jira if a link becomes available.